configurablecare/app/src/main/java/com/google/fhir/examples/configurablecare/care/CareConfiguration.kt

```diff
@@ -75,46 +74,17 @@ object ConfigurationManager {
     return bundleCollection
   }
 
-  fun setTaskConfigMap(planDefinitionId: String): MutableMap<String, String> {
-    val taskConfig =
+  fun setActiveRequestResourceConfiguration(planDefinitionId: String) {
```

Comment:
Saving this in the CareConfiguration class, which I think should be only readable, is not stateful. Closing the app and reopening it would lose all information of `activeRequestResourceConfiguration`. 
Now I think this might be the issue of why new app runs would require selecting PlanDefinition again otherwise it gave error.

Solution: we can store it in SharedPreferences for now.

Reply:
I agree that CareConfiguration should probably be readable only so `setActiveRequestResourceConfiguration()` and `getActiveRequestResourceConfiguration()` should be moved elsewhere. But isn't `setActiveRequestResourceConfiguration()` always run for the default PlanDefinition when the app is reopened? I don't see any error.

Reply:
Yes you are right!

So instead of storing the `activeRequestResourceConfig` in the CareConfiguration we can store it in WorkflowExecutionViewModel.

Reply:
Got it, will make that change!